### DISCUSSION TITLE
file inside object on a body request?

### DISCUSSION THREAD

**Discussion by rwxjon** [adonisjs/core#3992] [Help]
Hello there! 

I am sending a data structure like this

`{
files: [
      { key: 'key1', file: someFile },
      { key: 'key2', file: someFile },
     { key: 'key3', file: someFile }
 ]
}
`

but in adonis if I try to look at what I receive by doing request.all(), I am missing the files, is it possible to do what I am trying to do? I don't know if I'm skipping something

![imagen](https://user-images.githubusercontent.com/36550137/205092540-6c39b0ef-2930-4e7f-9bf2-bac929ee035a.png)

**Comment by Barbapapazes** [comment]
Hello,

Have you try this? https://docs.adonisjs.com/guides/file-uploads#accessing-uploaded-files

**Reply by rwxjon** [reply]
> Hello,
> 
> Have you try this? https://docs.adonisjs.com/guides/file-uploads#accessing-uploaded-files

I have tried but there is nothing, I have also checked by looking at the entire request, and I don't see any file

**Comment by galaczi** [comment]
Did you set ``enctype="multipart/form-data"``?

**Reply by rwxjon** [reply]
> Did you set `enctype="multipart/form-data"`?

Yep! I have tried with a little app i made and with postman, adding the header, and neither way.
